recipe/meta.yaml

```diff
@@ -30,8 +31,7 @@ requirements:
     - xorg-libxext
     - xorg-xextproto
   run:
-    - xorg-libx11
-    - xorg-libxext
+    - {{ pin_compatible('xorg-libx11') }}
```

Comment:
IIUC run exports should take care of these so no need to list them. Follows more up-to-date feedstocks like https://github.com/conda-forge/xorg-libxext-feedstock/blob/main/recipe/meta.yaml